app/models/concerns/property_summarisable.rb

```diff
@@ -2,24 +2,24 @@ module PropertySummarisable
   # MAIN HOME
   # Value
   def main_home_value
-    from_cfe_payload("assessment.capital.capital_items.properties.main_home.value") if owns_property?
+    from_cfe_payload("assessment.capital.capital_items.properties.main_home.value") || 0
```

Comment:
So it looks like the property fields all need an asylum support check, otherwise this fills in zeroes in a section that should be left blank.